optaplanner-score/optaplanner-score-drl/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-score</artifactId>
+    <version>8.17.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>optaplanner-score-drl</artifactId>
```

Comment:
Every module needs a proper name. See pom.xml of optaplanner-persistence-jpa to see which other properties might be required, such as the description element.

The description element in other poms isn't perfect, but let's keep this consistent and fix the description element's approach in a separate PR, for all pom.xml's.